### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by asylzhank** [vectorwing/FarmersDelight#1036] [Q&A]
Hi, guys
I want to translate the game to Kazakh, can I do that?

**Comment by asylzhank** [comment]
ok..  i figured it out
where can i find the template for it?

**Comment by vectorwing** [comment]
Hi! Feel free to translate Farmer's Delight to any language supported by Minecraft. 👍

You can depart from the 'en_us.json' lang file, as it's the main one. Then, once ready, you can open a Pull Request to have it merged on the `1.20` branch, or send me the .json directly.
